doc/developer/documenting.rst

```diff
@@ -34,4 +34,34 @@ documentation is built. To do this, you can clean the output with::
 	jupyter nbconvert --ClearOutputPreprocessor.enabled=True --inplace mynotebook.ipynb
 
 This should clear the output and overwrite the file. If this doesn't work, you can clear
-all output cells in the notebook web interface itself before committing the file.
\ No newline at end of file
+all output cells in the notebook web interface itself before committing the file.
+
+Documentation for ARMI plugins
+------------------------------
```

Comment:
Did you have something in mind to go here?

Reply:
It's just a section for subsections. Will add a line